mne/io/_digitization.py

```diff
@@ -201,7 +212,7 @@ def _foo_get_data_from_dig(dig):
             # XXX: point_names.append('HPI%03d' % d['ident'])
         elif d['kind'] == FIFF.FIFFV_POINT_EXTRA:
             hsp.append(d['r'])
-        elif d['kind'] == FIFF.FIFFV_POINT_EEG:
+        else:
```

Comment:
Any reason to change this? IIRC ECoG/SEEG etc. still get named `_EEG` in this structure

Reply:
Hmm not sure why I ended up changing this. Changed it back and it didn't break, so assuming it was a byproduct of an earlier iteration.